recipes/unidist/meta.yaml

```diff
@@ -0,0 +1,143 @@
+# from "unidist[<option>]" to "unidist-<option>" as a conda-forge package
+outputs:
+  - name: unidist-all
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        # unidist-core has multiprocessing and python backends by default
+        - {{ pin_subpackage('unidist-core', exact=True) }}
+        - {{ pin_subpackage('unidist-dask', exact=True) }}
+        # ray does not have OSX builds yet
+        - {{ pin_subpackage('unidist-ray', exact=True) }}   # [not osx]
+        # mpi4py does not have win32 and linux32
+        - {{ pin_subpackage('unidist-mpi', exact=True) }}   # [not win32 or not linux32]
+    test:
+      imports:
+        # dummy test; actual tests are in subpackages
+        - unidist
+
+  # compatibility with packagename before split into options
+  - name: unidist
+    requirements:
+      host:
+        - python
+      run:
+        - python
+    test:
+      imports:
+        - unidist
+
+  - name: unidist-core
+    build:
+      script: python -m pip install . --ignore-installed -vv
+    requirements:
+      host:
+        - pip
+        - python
+      run:
+        - python
+        - packaging
+        - cloudpickle
+    test:
+      imports:
+        - unidist
+      commands:
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('python'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('multiprocessing'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+
+  - name: unidist-dask
+    build:
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist-core', exact=True) }}
+        - dask >=2.22.0
+        - distributed >=2.22.0
+    test:
+      imports:
+        - unidist
+      commands:
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('dask'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+
+  - name: unidist-ray
+    build:
+      # There is no Ray in conda-forge for MacOS yet, see conda-forge/ray-packages-feedstock#2.
+      # There is no Ray for Python >3.9 for now.
+      # TODO: when Ray support for 3.10 is released, this restriction should be lifted.
+      skip: true  # [osx or py>39]
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist-core', exact=True) }}
+        - ray-default >=1.4.0
+        # aioredis 2.x breaks Ray
+        # ref: https://github.com/conda-forge/ray-packages-feedstock/issues/38
+        - aioredis <2
+    test:
+      imports:
+        - unidist
+      commands:
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('ray'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+
+  - name: unidist-mpi
+    build:
+      # There is no mpi4py in conda-forge for win32 and linux32 yet.
+      skip: true  # [win32 or linux32]
```

Comment:
We don't build for 32 bit on conda-forge, so this should be removed.

Reply:
Thanks for your review! I addressed comments and also excluded unidist-core build so that unidist is default.